### ISSUE TITLE
gcloud unrecognized arguments --fields

### ISSUE THREAD

**Issue by yujuhong** [kubernetes/kubernetes#25153]
https://console.cloud.google.com/storage/browser/kubernetes-jenkins/pr-logs/pull/25054/kubernetes-pull-build-test-e2e-gce/38324/

```
INSTANCE_GROUPS=e2e-gce-master-1-minion-group
NODE_NAMES=e2e-gce-master-1-minion-0pce e2e-gce-master-1-minion-3mfi e2e-gce-master-1-minion-4jgj e2e-gce-master-1-minion-bmu8 e2e-gce-master-1-minion-qirl e2e-gce-master-1-minion-vywd
usage: gcloud compute instances describe  NAME [optional flags]
ERROR: (gcloud.compute.instances.describe) unrecognized arguments:
  --fields
  networkInterfaces[0].accessConfigs[0].natIP
```

**Comment by yujuhong** [comment]
I believe all per-PR builds failed because of the same reason.

**Comment by spxtr** [comment]
It's not a flake, it's just plain broken. Assigning to @ixdy, test infra oncall.

**Comment by ixdy** [comment]
There appears to be a new gcloud release today: https://cloud.google.com/sdk/release_notes

Not sure if this was an expected breaking change or not. I don't see it in the release notes, anyway.

**Comment by a-robinson** [comment]
cc @k8s-oncall as well

**Comment by ixdy** [comment]
Reassigning to the buildcop. I'm pretty sure this isn't an infra issue - OSS k8s is just broken with the latest release of gcloud.

**Comment by ixdy** [comment]
Apparently `--fields` was an undocumented hidden flag that we should not have used. We are to replace the `--fields` and `--format` flags with
`--format="value(networkInterfaces[0].accessConfigs[0].natIP)`

We'll probably need to cherry-pick this fix.

**Comment by ixdy** [comment]
Fix PR: #25159
